ampersand-multifield-view.js

```diff
@@ -73,9 +98,25 @@ var MultiFieldView = View.extend({
   },
 
   isValid: function() {
-    return this.fields.every(function(field) {
+    var fieldsValid =  this.fields.every(function(field) {
       return field.valid;
     });
+
+    var valid = !this.runTests();
+
+    return fieldsValid && valid;
+  },
+
+  runTests: function() {
+    var errMsg = '';
+
+    this.tests.some(function(test) {
+      errMsg = test.call(this, this.value) || '';
```

Comment:
since you're already importing `lodash.find`, you could also do something like:
```js
runTests: function() {
  this.message = find(this.tests, function(test) {
    return test.call(this, this.value);
  }, this) || '';

  return this.message;
}
```
it's fine as is though, if you don't like the above